src/github/get-diff-from-commit.ts

```diff
@@ -13,16 +13,27 @@ export type ReturnTypeOfDiffFiles = {
 export const getFileDiffFromGithub = async ({
   base,
   head
-}: DiffFilesType): Promise<ReturnTypeOfDiffFiles> => {
+}: DiffFilesType, page = 1, allFiles = []): Promise<ReturnTypeOfDiffFiles> => {
   console.log(`fetching file diff for ${base} ${head}`)
+  
   const resp = await github.client.rest.repos.compareCommits({
```

Comment:
```suggestion
}: DiffFilesType, page = 1): Promise<ReturnTypeOfDiffFiles> => {
  console.log(`fetching file diff for ${base} ${head}`)
  const allFiles = []
  const resp = await github.client.rest.repos.compareCommits({
```

Reply:
How would this work? That argument is passed in when recursively called.

Reply:
Ok. I actually completely changed my approach.